test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
Having an image namespace common to all tests doesn't seem like too bad of
an idea, especially if we can come up with a good one.  Then we can have
"run.sh" optionally clean up all images matching "ournamespace/*" (or just
not worry about it, if we can come up with an obvious enough name for the
namespace).